plugins/MultiSites/vue/src/AllWebsitesDashboard/KPICard.vue

```diff
@@ -7,35 +7,37 @@
 
 <template>
   <div class="kpiCard">
-    <div class="kpiCardTitle">
-      <span :class="`kpiCardIcon ${kpi.icon}`" />
-      {{ translate(kpi.title) }}
-    </div>
+    <div class="kpiCardContent">
```

Comment:
This element is used to have the table cells in `.kpiCardEvolution` not have equal width, right?

I think the trick used to collapse the table sparkline cell (`.sitesTableSparkline`) could also work here:

```css
.kpiCardEvolutionTrend {
  width: 1%;
}
```

The column should still stretch according to accommodate the content, but otherwise take up the least amount of space. And that should make the `.kpiCardContent` container superfluous.

Or is there any screensize/i18n combination known where this would break without the extra container?

Reply:
Hm. That might work as well. I'll do some testing on that...